pkg/square/builder_test.go

```diff
@@ -262,178 +161,186 @@ func rawData(shares []shares.Share) ([]byte, error) {
-			),
-			expectedIndexes: [][]uint32{{3}, {5, 8}, {4}},
-		},
-		{
-			// no blob txs should make it in the square
-			squareSize: 1,
-			blobTxs: generateBlobTxsWithNamespaces(
-				t,
-				[]ns.Namespace{ns1, ns2, ns3},
-				[][]int{{1000}, {1000}, {1000}},
-			),
-			expectedIndexes: [][]uint32{},
-		},
-		{
-			// only two blob txs should make it in the square (after reordering)
-			squareSize: 4,
-			blobTxs: generateBlobTxsWithNamespaces(
-				t,
-				[]ns.Namespace{ns3, ns2, ns1},
-				[][]int{{2000}, {2000}, {5000}},
-			),
-			expectedIndexes: [][]uint32{{7}, {2}},
-		},
-		{
-			squareSize: 4,
-			blobTxs: generateBlobTxsWithNamespaces(
-				t,
-				[]ns.Namespace{ns3, ns3, ns2, ns1},
-				[][]int{{1800, 1000}, {22000}, {1800}},
-			),
-			// should be ns1 and {ns3, ns3} as ns2 is too large
-			expectedIndexes: [][]uint32{{6, 10}, {2}},
-		},
-		{
-			squareSize: 4,
-			blobTxs: generateBlobTxsWithNamespaces(
-				t,
-				[]ns.Namespace{ns1, ns3, ns3, ns1, ns2, ns2},
-				[][]int{{100}, {1400, 900, 200, 200}, {420}},
-			),
-			expectedIndexes: [][]uint32{{3}, {7, 10, 4, 5}, {6}},
-		},
-		{
-			squareSize: 4,
-			blobTxs: generateBlobTxsWithNamespaces(
-				t,
-				[]ns.Namespace{ns1, ns3, ns3, ns1, ns2, ns2},
-				[][]int{{100}, {900, 1400, 200, 200}, {420}},
-			),
-			expectedIndexes: [][]uint32{{3}, {7, 9, 4, 5}, {6}},
-		},
-		{
-			squareSize: 16,
-			blobTxs: generateBlobTxsWithNamespaces(
-				t,
-				[]ns.Namespace{ns1, ns1},
-				[][]int{{100}, {shares.AvailableBytesFromSparseShares(appconsts.DefaultSubtreeRootThreshold)}},
-			),
-			// There should be one share padding between the two blobs
-			expectedIndexes: [][]uint32{{2}, {3}},
-		},
-		{
-			squareSize: 16,
-			blobTxs: generateBlobTxsWithNamespaces(
-				t,
-				[]ns.Namespace{ns1, ns1},
-				[][]int{{100}, {shares.AvailableBytesFromSparseShares(appconsts.DefaultSubtreeRootThreshold) + 1}},
-			),
-			// There should be one share padding between the two blobs
-			expectedIndexes: [][]uint32{{2}, {4}},
-		},
-	}
-	for i, tt := range tests {
-		t.Run(fmt.Sprintf("case%d", i), func(t *testing.T) {
-			builder, err := square.NewBuilder(tt.squareSize, appconsts.LatestVersion)
-			require.NoError(t, err)
-			for _, tx := range tt.blobTxs {
-				blobTx, isBlobTx := blob.UnmarshalBlobTx(tx)
-				require.True(t, isBlobTx)
-				_ = builder.AppendBlobTx(blobTx)
-			}
-			square, err := builder.Export()
-			require.NoError(t, err)
-			txs, err := shares.ParseTxs(square)
-			require.NoError(t, err)
-			for j, tx := range txs {
-				wrappedPFB, isWrappedPFB := coretypes.UnmarshalIndexWrapper(tx)
-				require.True(t, isWrappedPFB)
-				require.Equal(t, tt.expectedIndexes[j], wrappedPFB.ShareIndexes, j)
-			}
-		})
-	}
-}
+// // TestSquareBlobPositions ensures that the share commitment rules which dictate the padding
+// // between blobs is followed as well as the ordering of blobs by namespace.
+// func TestSquareBlobPostions(t *testing.T) {
```

Comment:
[blocking] is this intended to be committed commented out? Instead of merging commented code can we:

delete if no longer relevant
uncomment and fix
worst case: delete with a GH issue to re-add?